manager/config/movement-controller/kustomization.yaml

```diff
@@ -9,7 +9,7 @@ namespace: m4d-system
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: m4d
+namePrefix: m4d-
```

Comment:
why the change?

Reply:
It was an old inconsistency that I fixed. This is the kustomize folder that I use for the motion only testing.